taf/auth_repo.py

```diff
@@ -236,8 +242,20 @@ def sorted_commits_and_branches_per_repositories(
             *commits, target_repos=target_repos, default_branch=default_branch
         )
         previous_commits = {}
+        skipped_targets = []
+        excluded_target_globs = excluded_target_globs or []
         for commit in commits:
             for target_path, target_data in targets[commit].items():
+                if target_path in skipped_targets:
+                    continue
```

Comment:
Question: Is this check needed?  
If `skip_target` is True, L257 will `continue` to next target

Current approach makes sense if we move the [for loop](https://github.com/openlawlibrary/taf/pull/250/files#diff-4b51c03cf2230aad079541d7d975a521e6a9299c0bd717ed0952065f7288b172R252-R255) before loop `repositories.items()`